org.unofficialrevport.REVHubInterface.yml

```diff
@@ -0,0 +1,69 @@
+id: org.unofficialrevport.REVHubInterface
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: REVHubInterface.sh
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak 
```

Comment:
why do you need this dbus access? It's not allowed by default.

Reply:
In order to communicate with the REV Hub (What the program is intended to communicate and operate with)

Reply:
can't it do that without opening a hole?

Reply:
fixed

Reply:
again your lack of explanation make it hard to understand what you are not telling.

Reply:
It was from testing, I forgot to remove it, same with doc:

Reply:
> again your lack of explanation make it hard to understand what you are not telling.

when I was making the flatpak file for this I based it off of the one from telegram desktop because it was also built from python and forgot to remove it's other permissions, sorry